os_builders/roles/vm_baseline/tasks/qemu-guest-agent.yml

```diff
@@ -0,0 +1,14 @@
+- name: Install Qemu Guest Agent on Ubuntu
+  block:
+    - name: Install Alien for Pakiti
+      apt:
+        name: qemu-guest-agent
+        state: present
+        update_cache: yes
+  when: ansible_distribution == "Ubuntu"
```

Comment:
Does this need to be a block if it is one task?

Reply:
It doesnt need to be, removed now